tests/internal/params/vars.go

```diff
@@ -1,9 +1,17 @@
 package params
 
+import "github.com/golang/glog"
+
 // PrivilegedNSLabels represents privileged labels.
-var PrivilegedNSLabels = map[string]string{
-	"pod-security.kubernetes.io/audit":               "privileged",
-	"pod-security.kubernetes.io/enforce":             "privileged",
-	"pod-security.kubernetes.io/warn":                "privileged",
-	"security.openshift.io/scc.podSecurityLabelSync": "false",
-}
+var (
+	PrivilegedNSLabels = map[string]string{
+		"pod-security.kubernetes.io/audit":               "privileged",
+		"pod-security.kubernetes.io/enforce":             "privileged",
+		"pod-security.kubernetes.io/warn":                "privileged",
+		"security.openshift.io/scc.podSecurityLabelSync": "false",
+	}
+	// LogLevel is the verbosity for global tools.
+	LogLevel glog.Level = 90
```

Comment:
It's OK to duplicate this LogLevel no need to move it to global vars.

Reply:
fixed